devito/base.py

```diff
@@ -39,6 +40,16 @@ class SparseTimeFunction(with_metaclass(_BackendSelector, function.SparseTimeFun
     pass
 
 
+class PrecomputedSparseFunction(with_metaclass(_BackendSelector,
+                                               function.PrecomputedSparseFunction)):
+    pass
+
+
+class PrecomputedSparseTimeFunction(with_metaclass(_BackendSelector,
+                                                   PrecomputedSparseTimeFunction)):
```

Comment:
just function.PrecomputedSparseTimeFunction no? like all the others . Or is there a problem with that?

Reply:
Flake cries about line length.